src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+
+        Figure[] figures = new Figure[6];
+        for (int i = 0; i < figures.length; i++) {
+            if (i >= figures.length / 2) {
+
+                figures[i] = figureSupplier.getDefaultFigure();
+
+            } else {
+
+                figures[i] = figureSupplier.getRandomFigure();
+
+            }
+        }
+
+        for (Figure figure : figures) {
+            figure.draw();
+        }
```

Comment:
1. Please do not use blank lines inside logical code block, we may sometimes use blank lines between them to separate one idea from another but here it's redundant
2. We may use 1 loop here
```suggestion
        for (int i = 0; i < figures.length; i++) {
            if (i >= figures.length / 2) {
                figures[i] = figureSupplier.getDefaultFigure();
            } else {
                figures[i] = figureSupplier.getRandomFigure();
            }
            figures[i].draw();
        }
```